README.md

```diff
@@ -322,3 +322,44 @@ You can decode custom types the same way, as long as the type also conforms to
 `JSONDecodable`.
 
 For more examples on how to use Argo, please check out the tests.
+
+Contributing
+------------
+
+See the [CONTRIBUTING] document.
+Thank you, [contributors]!
+
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/Argo/graphs/contributors
+
+Need Help?
+----------
+
+We offer 1-on-1 coaching. We can help you with functional programming in Swift,
+get started writing integration tests, and convert from Objective C to Swift.
+[Get in touch].
+
+[Get in touch]: http://coaching.thoughtbot.com/rails/?utm_source=github
+
+License
+-------
+
+Argo is Copyright (c) 2015 thoughtbot, inc. It is free software, and may be
+redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
+About
+-----
+
+![thoughtbot](https://thoughtbot.com/logo.png)
+
+Argo is maintained and funded by thoughtbot, inc.
+The names and logos for thoughtbot are trademarks of thoughtbot, inc.
+
+We love open source software!
+See [our other projects][community]
+or [hire us][hire] to help build your product.
```

Comment:
Since we don't have a page for our iOS case studies -- and to reduce the churn that linking to the specific ones would cause -- I'm going to link to /ios.